src/main/webapp/app/lecture/pdf-preview/pdf-preview.component.html

```diff
@@ -0,0 +1,32 @@
+<div class="row justify-content-center">
+    <div class="col-12">
+        <div>
+            <h4>
+                <span jhiTranslate="artemisApp.attachment.pdfPreview.title"></span>
+                @if (attachment) {
+                    {{ attachment.id }}: {{ attachment?.name }}
+                }
+                @if (attachmentUnit) {
+                    {{ attachmentUnit.id }}: {{ attachmentUnit.name }}
+                }
+            </h4>
+            <div class="row">
+                <div class="pdf-container" #pdfContainer>
+                    @if (isEnlargedView) {
+                        <div class="enlarged-container" (click)="closeIfOutside($event)">
+                            <canvas #enlargedCanvas></canvas>
+                            <button class="btn btn-close" (click)="closeEnlargedView(); $event.stopPropagation()"></button>
+                            @if (currentPage !== 1) {
+                                <button class="btn btn-secondary nav-button left" (click)="navigatePages('prev'); $event.stopPropagation()">←</button>
+                            }
+                            @if (currentPage !== totalPages) {
+                                <button class="btn btn-secondary nav-button right" (click)="navigatePages('next'); $event.stopPropagation()">→</button>
```

Comment:
Could you define a common method that handles the navigation & calls `stopPropagation` on the event? That way, you won't have to add it to each button here

Reply:
Done